src/content/docs/pt-br/community-resources/talks.mdx

```diff
@@ -0,0 +1,48 @@
+---
+title: Palestras, Entrevistas, e Transmissões
+description: Assista palestras, entrevistas, e transmissões sobre Astro!
+i18nReady: true
+---
+
+import Badge from '~/components/Badge.astro';
+
+Aqui estão alguns dos nossos vídeos favoritos da comunidade para introduzir você ao Astro!
+
+Para mais conteúdo sobre Astro, confira [o Blog do Astro](https://astro.build/blog/) ou assine [a newsletter oficial do Astro](https://astro.build/newsletter/signup) onde nós trazemos o melhor conteúdo da comunidade de cada mês.
+
+:::tip
+Se inscreva [no calendário de Eventos da Comunidade Astro no Google](https://calendar.google.com/calendar/embed?src=c_g6re3u5gkl0mug3ktn1dibqhfk%40group.calendar.google.com) e nunca perca um evento ao vivo!
+:::
+
+## Sobre o Astro
+
+- [Astro em 100 Segundos](https://www.youtube.com/watch?v=dsTXcSeAZq8) por Fireship
+- [Porque eu vou escolher Astro (quase) sempre em 2024](https://www.youtube.com/watch?v=kssIEqSJeMI) por Jason Lengstorf
+- [Eu experimentei Astro e AMEI - 5 razões que você vai também!](https://www.youtube.com/watch?v=wND4lSml31A) por James Q Quick
+- [O que é Astro?](https://www.youtube.com/watch?v=oQDErNNzdwU) por Kodaps Academy
+- [Tudo que você precisa saber sobre o Astro](https://www.youtube.com/watch?v=rRxuVOutmFQ) por @awesome
+- [Arquitetura em Ilhas | Astro Ilhas explicado de forma simples](https://www.youtube.com/watch?v=kwqlP6Pe0SI) por ExcaliBear Codes
+- [Astro é um divisor de águas para desenvolvedores Nuxt e Vue?](https://www.youtube.com/watch?v=Sq7eKxPqfSo) por Program with Erik
+- [Como Astro fez meu site 100x mais rápido](https://www.youtube.com/watch?v=cOxA3kMYtkM) por James Perkins
+
+## Palestras em Conferências
+
+- [Transições de Visualização: Fato vs. Ficção](https://www.youtube.com/watch?v=iT-3amHK7tA&list=PLqGQbXn_GDmkOsHI7-Wrbv1GgAA4tJZhg&index=29)  por Fred Schott na ViteConf 2023
+- [Trazendo React para o seu site de conteúdo com Astro](https://www.youtube.com/watch?v=Fouws8iKPKQ) por Matthew Phillips na Future Frontend 2023
+- [Tudo sobre Astro](https://www.youtube.com/watch?v=WKXHFIulZ9g) por Elian Van Cutsem, React Day Berlin 2023
+
+## Entrevistas e podcasts
+
+- [Astro torna os frameworks pesados de JS obsoletos?](https://www.youtube.com/watch?v=4Ohi9G-RCcE) com Fred K. Schott no Learn with Jason
+- [Papo sobre tudo sobre Astro](https://www.youtube.com/watch?v=3eyedOP83fw) com Matthew Phillips no WebStorm Webinar
+- [Decodificando Dados de Desenvolvedor: Analisando o relatório Web Dev | Qual é o framework número um para 2024?](https://youtu.be/2vqIHyz6iKs?si=vLcyzm07teC6v7u5&t=1679) com Sterling Davis e Matt Kane da Netlify
+- [Snowpack, Astro](https://www.devtools.fm/episode/14) com Fred K. Schott no devtools.fm
+- [Astro e MDX para seu jardim digital](https://podrocket.logrocket.com/astro-mdx-kathleen-mcmahon) com Kathleen McMahon no PodRocket
```

Comment:
This was the first time that I saw the term "digital gardening". I made some researches in the internet, and seems that the literal translation is the correct one.